### PR TITLE
Task 94 fulfillment skus article

### LINKED ISSUE #94: Write documentation for digital products

### PR DESCRIPTION
**Pull request by mttdffy**
closes #94

### CODE REVIEW

File: docs/source/articles/fulfillment-skus.html.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+title: Fulfillment SKUs
+created_at: 2019/10/02
+excerpt: Define how products in your catalog are fulfilled. Fulfillment SKUs allow the automation of digital and similar items.
+---
+
+# Fulfillment SKUS
```
Comment (bencrouse) [inline docs/source/articles/fulfillment-skus.html.md]:
```suggestion
# Fulfillment SKUs
```

File: docs/source/articles/fulfillment-skus.html.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+The `ignore` policy is a simple no-op policy that takes no action on items ordered and does not require shipping.
+
+The `ship` policy requires shipping, but takes no automated action when a SKU with this policy is ordered. **This is the default policy used when no policy exists for a SKU.**
+
+The `download` policy will automatically generate a `Fulfillment::Token` for a customer when they purchase a SKU with that policy. This token provides a unique URL to the customer that is specific to their order and account. This will allow them to download the `file` tied to the Fulfillment SKU from the provided link on the order confirmation page, in the confirmation email, and/or in their order history.  
+
+## Fulfillment Tokens
+
+A `Fulfillment::Token` is a documented associated to the Fulfillment SKU that stores a secure random string. This association allows a URL with a unique token to be used to provide access to the `file` on the `Fulfillment::SKU` document. Tokens will also be associated to a user and an order if generated as a result of a customer placing an order. Tokens can also be generated via the admin UI.
```
Comment (chriscressman) [inline docs/source/articles/fulfillment-skus.html.md]:
```suggestion
A `Fulfillment::Token` is a document associated to the Fulfillment SKU that stores a secure random string. This association allows a URL with a unique token to be used to provide access to the `file` on the `Fulfillment::SKU` document. Tokens will also be associated to a user and an order if generated as a result of a customer placing an order. Tokens can also be generated via the admin UI.
```

File: docs/source/articles/fulfillment-skus.html.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
+title: Fulfillment SKUs
+created_at: 2019/10/02
+excerpt: Define how products in your catalog are fulfilled. Fulfillment SKUs allow the automation of digital and similar items.
+---
+
+# Fulfillment SKUS
+
+Workarea v3.5 adds the concept of Fulfillment SKUs -- a model that defines policies for delivering ordered products to customers. This provides a more robust mechanism for the fulfillment of digital items while providing the flexibility to offer customized delivery behaviors at the SKU level.
```
Comment (chriscressman) [inline docs/source/articles/fulfillment-skus.html.md]: the term you're defining is plural, but the definition is singular; should make those consistent

Comment (mttdffy) [inline docs/source/articles/fulfillment-skus.html.md]:
```suggestion
Workarea v3.5 adds the concept of a Fulfillment SKU -- a model that defines policies for delivering ordered products to customers. This provides a more robust mechanism for the fulfillment of digital items while providing the flexibility to offer customized delivery behaviors at the SKU level.
```

**Pull request merged**